### PR TITLE
Update proj4 docs

### PR DESCRIPTION
**Pull request by kylebarron**

### CODE REVIEW

File: modules/proj4/docs/api-reference/proj4-projection.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
 # Ellipsoid
```
Comment (ibgreen) [inline modules/proj4/docs/api-reference/proj4-projection.md:1]: Whoops Title is wrong

File: modules/proj4/docs/api-reference/proj4-projection.md [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
 
-
 ## Usage
 
-Reproject WGS84 coordinates in another CRS
+Reproject WGS84 coordinates to another CRS
```
Comment (ibgreen) [inline modules/proj4/docs/api-reference/proj4-projection.md:5]: From CRS to CRS not just WGS84

Comment (kylebarron) [reply modules/proj4/docs/api-reference/proj4-projection.md:5]: Here the example is from WGS84, I thought it would be clear enough to a user

Comment (ibgreen) [reply modules/proj4/docs/api-reference/proj4-projection.md:5]: Ah yes, makes sense, please disregard my comment.

**Pull request merged**